src/dmd/root/stringtable.d

```diff
@@ -103,6 +103,17 @@ public:
         pools = null;
     }
 
+    /**
+     * Looks up the given string in the string table and returns its associated
+     * value.
+     *
+     * Params:
+     *  s = the string to look up
+     *  length = the length of $(D_PARAM s)
+     *
+     * Returns: the string's associated value, or `null` if the string doesn't
+     *  exist in the string table
+     */
```

Comment:
FWIW the DStyle we use for new symbols at Druntime/Phobos says:

> Documentation comments should not have leading stars on each line.

https://dlang.org/dstyle.html#phobos_documentation

Reply:
😞

Reply:
It looks so weird. I have a feeling that most of the code out there is not using this style.

Reply:
Fixed.